src/main/java/duke/commands/Task.java

```diff
@@ -0,0 +1,31 @@
+package duke.commands;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String toString(){
+        return "[" + this.getStatusIcon() + "] " + this.description;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "\u2713" : "\u2718"); //return tick or X symbols
```

Comment:
you might want to store this in a variable

Reply:
just to clarify, do you want me to store the tick and cross as variables? I'll do that. :)